apollo-test/src/main/java/com/spotify/apollo/test/ServiceHelper.java

```diff
@@ -478,7 +480,18 @@ private void start(int timeoutSeconds, final String... args) throws InterruptedE
             .withModule(HttpClientModule.create())
             .withModule(
                 ForwardingStubClientModule
-                    .create(forwardNonStubbedRequests, stubClient.asRequestAwareClient()));
+                    .create(forwardNonStubbedRequests, stubClient.asRequestAwareClient()))
+            .withModule(new AbstractApolloModule() {
```

Comment:
Done - but I'm starting to feel more and more that we should separate out the Meta module from the environment module. A problem with that is that ApolloEnvironmentImpl initialises the metadata collection. But that would probably be OK - it means that ApolloEnvironmentModule requires MetaModule to work, and any assembly using it has to also use the MetaModule?